app/navigation/houseConsignment/index/items/HouseConsignmentItemNavigator.scala

```diff
@@ -66,8 +73,12 @@ class HouseConsignmentItemNavigator(houseConsignmentMode: Mode) extends Navigato
       ua => Some(controllers.routes.HouseConsignmentController.onPageLoad(ua.id, houseConsignmentIndex))
     case AddNetWeightYesNoPage(houseConsignmentIndex, itemIndex) =>
       ua => addNetWeightYesNoRoute(ua, ua.id, houseConsignmentIndex, itemIndex, CheckMode)
-    case NetWeightPage(houseConsignmentIndex, _) =>
-      ua => Some(controllers.routes.HouseConsignmentController.onPageLoad(ua.id, houseConsignmentIndex))
+    case NetWeightPage(houseConsignmentIndex, itemIndex) =>
+      ua => netWeightPageRoute(ua, CheckMode, houseConsignmentIndex, itemIndex)
+    case UniqueConsignmentReferenceYesNoPage(houseConsignmentIndex, itemIndex) =>
```

Comment:
The `UniqueConsignmentReferenceYesNoPage` case is redundant, as the page can't be accessed in CheckMode for the item mode